resources/views/manager/profile/about.html.twig

```diff
@@ -138,12 +106,21 @@
                             {% for department in departments %}
                                 {% set deparmentName = department.name %}
                                 {% set chosenName = manager.department.name %}
-                                <option
-                                    value="{{ department.id }}"
-                                    {% if deparmentName == chosenName %}selected="selected"{% endif %}
-                                    >{{ department.name }}</option>
+                                    <option
```

Comment:
This is super nitpicky, but while we're here, the the indentation in this block is a bit off :P